vendor/github.com/contiv/libOpenflow/util/stream.go

```diff
@@ -35,6 +37,8 @@ type MessageStream struct {
 	pool *BufferPool
 	// Message parser
 	parser Parser
+	// Channel to shut down the parser goroutine
+	parserShutdown chan bool
```

Comment:
@rchirakk The comments on that PR should show exactly where that happens. The outbound message processing goroutine has a select which watches the shutdown channel, in addition to watching for outbound messages to be sent out. We already had the means to shut down that outbound message processing queue. That PR has added a for loop which tells the individual goroutines which process inbound messages to shut down - just as was done before for the outbound messages goroutine. This means that the main shutdown procedure of this message sending & receiving code triggers the shutdown of the inbound message processing goroutines the same way the outbound message queue was shut down before.

This has no impact on performance or on how the code works in the end. If that code had been broken or its behavior had been altered in any way which caused issues, I would've discovered that.